public/app/core/components/AppNotifications/state/actions.ts

```diff
@@ -0,0 +1,28 @@
+import { AppNotification } from 'app/types';
+
+export enum ActionTypes {
+  AddAppNotification = 'ADD_APP_NOTIFICATION',
+  ClearAppNotification = 'CLEAR_APP_NOTIFICATION',
+}
+
+interface AddAppNotificationAction {
```

Comment:
need to move these actions to app/core/actions/appNotifications.ts where we have more shared / global actions  (and exported via app/core/actions/index.ts